mlflow/tracking/client.py

```diff
@@ -665,10 +704,41 @@ def search_model_versions(self, filter_string):
         """
         Search for model versions in backend that satisfy the filter criteria.
 
-        :param filter_string: A filter string expression. Currently supports a single filter
-                              condition either name of model like ``name = 'model_name'`` or
+        :param filter_string: A filter string expression. Currently, it supports a single filter
+                              condition either a name of model like ``name = 'model_name'`` or
                               ``run_id = '...'``.
         :return: PagedList of :py:class:`mlflow.entities.model_registry.ModelVersion` objects.
+
+        .. code-block:: python
+            :caption: Example
+
+            import mlflow
+
+            model_name = "CordobaWeatherForecastModel"
+            run_id = "e14afa2f47a040728060c1699968fd43"
+            filter_name = "name='{}'".format(model_name)
+            filter_runid = "run_id='{}'".format(run_id)
+            client = mlflow.tracking.MlflowClient()
+
+            # Get all versions of the model filtered by name
+            results = client.search_model_versions(filter_name)
+            print("-" * 80)
+            for res in results:
+                print("name={}; run_id={}; version={}".format(res.name, res.run_id, res.version))
+            # Get the version of the model filtered by run_id
+            results = client.search_model_versions(filter_runid)
+            print("-" * 80)
+            for res in results:
+                print("name={}; run_id={}; version={}".format(res.name, res.run_id, res.version))
```

Comment:
```suggestion
            client = mlflow.tracking.MlflowClient()

            # Get all versions of the model filtered by name
            model_name = "CordobaWeatherForecastModel"
            filter_name = "name='{}'".format(model_name)
            results = client.search_model_versions(filter_name)
            print("-" * 80)
            for res in results:
                print("name={}; run_id={}; version={}".format(res.name, res.run_id, res.version))

            # Get the version of the model filtered by run_id
            run_id = "e14afa2f47a040728060c1699968fd43"
            filter_runid = "run_id='{}'".format(run_id)
            results = client.search_model_versions(filter_runid)
            print("-" * 80)
            for res in results:
                print("name={}; run_id={}; version={}".format(res.name, res.run_id, res.version))
```

Reply:
Just make the code more readable

Reply:
Indeed, makes it readable. Will change it.

Reply:
done and committed all feedback.